pkgs/tools/graphics/epub-thumbnailer/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, python3Packages, fetchFromGitHub}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "epub-thumbnailer";
+  version = "unstable-2022-04-22";
```

Comment:
Im not sure how unstable versioning should work but I always put the date of the commit. Which would be 2022-02-10 if you grab the latest

Reply:
hm you're right, I didn't know.

Reply:
Fixed!